virttest/block.py

```diff
@@ -0,0 +1,22 @@
+"""
+Utility functions to handle block related actions.
+
+"""
+
+from . import qemu_monitor
+
+
+def get_block_virtual_size(dev_dict):
+    """
+    Get block virtual-size of specified block device
+    from monitor's info block command.
+
```

Comment:
From the code, I believe you want to use qmp but not hmp, so pls describe it here.

Reply:
Get block virtual-size of specified block device with dev_name  ***from the qemu monitor***

Reply:
is you mean from qmp monitor?